libtiledbvcf/src/read/reader.cc

```diff
@@ -183,6 +183,10 @@ void Reader::init_af_filter() {
     Group group(*ctx_, dataset_->root_uri(), TILEDB_READ);
     af_filter_ = std::make_unique<VariantStatsReader>(ctx_, group);
     af_filter_->set_condition(params_.af_filter);
+    // TODO: enable sorting only if needed for GVCF IAF feature
```

Comment:
Is this `TODO` done?

Reply:
There's currently a check for the V3 stats array being in use; it might be more efficient for queries against datasets with V3 variant stats, that don't request IAC, IAN, or IAF not to enable sorting; this check isn't performed yet.